Youtube.py

```diff
@@ -1,17 +1,17 @@
 import pafy
 
-url= raw_input("Enter your url")
+url= raw_input("Enter your url: ")
 video=pafy.new(url)
-print "Do you Want to Download it or want to extract information ?"
-choice = raw_input("Input d to Download or ei to Extract Information")
+print "Do you Want to Download it or want to extract information?"
+choice = raw_input("Input 'download' to Download or 'extract' to Extract Information: ")
 
-if choice == 'd':
+if choice == 'download':
 	best=video.getbest()
 	best.download(quiet=False)
-elif choice == 'ei':
-	print "Title : " +video.Title
+elif choice == 'extract':
```

Comment:
@sehgalayush1 I think using `d` and `ei` will be easy for the user to use instead of `download` and `extract`. Rest looks good.